File: ios/WonderPushNotificationServiceExtension/NotificationService.m
//
//  NotificationService.m
//  WonderPushNotificationServiceExtension
//
//  Created by VZ on 16/4/25.
//

#import "NotificationService.h"

@implementation NotificationService

+ (NSString *)clientId {
    NSString *clientId = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"WONDERPUSH_CLIENT_ID"];
    return clientId ?: @"";
}

+ (NSString *)clientSecret {
    NSString *clientSecret = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"WONDERPUSH_CLIENT_SECRET"];
    return clientSecret ?: @"";
}

@end

File: ios/WonderPushNotificationServiceExtension/NotificationService.swift
import WonderPushExtension
import Foundation

class NotificationService: WPNotificationServiceExtension {

    override class func clientId() -> String {
        return Bundle.main.object(forInfoDictionaryKey: "WONDERPUSH_CLIENT_ID") as? String ?? ""
    }

    override class func clientSecret() -> String {
        return Bundle.main.object(forInfoDictionaryKey: "WONDERPUSH_CLIENT_SECRET") as? String ?? ""
    }

}
